src/store/modules/groups.js

```diff
@@ -77,6 +76,11 @@ export const getters = {
     return state.idsList.map(getters.get)
   },
   status: state => ({ isWaiting: state.isWaiting, error: state.error }),
+  saveError: (state, getters) => {
+    // access meta error getter with metaID
+    // a bit cumbersome!
+    return getters['meta/error'](`save-${getters.activeGroupId}`)
```

Comment:
Maybe we should revisit the idea of multiple meta modules if there are multiple data types, so each meta is just the object id, or a string.

In this case though is there any reason not to just make the meta id the group id?

Reply:
To keep the existing behaviour, I would use IDs like `save`, `join`, `leave`, `create`... there seems currently no need to handle the status of multiple groups at the same time.
If we would need the latter, than having multiple meta modules makes sense (namespaced by action?)